### PR TITLE
Add Iran rust meetup event

### PR DESCRIPTION
**Pull request by hamidrezakp**

### CODE REVIEW

**Comment by mariannegoldin** [comment]
Thank you for the PR! The link included points to an older meetup. Is there another link you would prefer to use for the 8/18/22 event?

**Comment by hamidrezakp** [comment]
> 
 oh, the link is correct, we had a problem with last meetup and it didn't started, so we are using that number :))

**Comment by hamidrezakp** [comment]
I have updated the meetup website, so it's all correct now

File: draft/2022-08-10-this-week-in-rust.md [hunk 1]
```diff
@@ -112,6 +112,8 @@ Rusty Events between 2022-08-10 - 2022-09-07 🦀
     * [**Hierarchical Task Network compiler written in Rust**](https://www.meetup.com/charlottesville-rust-meetup/events/287203159/)
 * 2022-08-18 | Virtual (Stuttgart, DE) | [Rust Community Stuttgart](https://www.meetup.com/Rust-Community-Stuttgart/)
     * [**Rust-Meetup**](https://www.meetup.com/rust-community-stuttgart/events/qtvtvsydclbxb/)
+* 2022-08-18 | Virtual (Tehran, IR) | [Iran Rust Meetup](https://rust-meetup.ir/)
+    * [**Iran Rust Meetup #8**](https://rust-meetup.ir/2022/07/22/8th-meetup.html)
```
Comment (andrewpollack) [inline draft/2022-08-10-this-week-in-rust.md:116]:
```suggestion
    * [**Iran Rust Meetup #8**](https://rust-meetup.ir/2022/08/18/8th-meetup.html)
```
URL change needed for date

**Review by andrewpollack — CHANGES_REQUESTED** [review]
Once this change is committed, this will be good to merge!

**Comment by hamidrezakp** [comment]
> Once this change is committed, this will be good to merge!

You are absolutely right, thanks for seeing my error :))

**Comment by andrewpollack** [comment]
Looks great; thank you!!

**Pull request merged**